### PR TITLE
Adding max execution time to PHP provisioner

### LINKED ISSUE #77: dev/build times out
I suggest raising the max_execution_time to 90 seconds.

### PR DESCRIPTION
**Pull request by dhensby**
Fixes #77 

Add an argument for PHP provisioner to increase max exec time

### CODE REVIEW

File: scripts/php.sh [hunk 1]
```diff
@@ -41,6 +44,7 @@ function usage() {
 	-q Quient mode
```
Comment (willmorgan) [inline scripts/php.sh]: Any chance this typo can be resolved? Boy scout rule and all that. Thanks!

File: scripts/php.sh [hunk 2]
```diff
@@ -56,6 +60,7 @@ function run() {
 	cp -f /usr/share/doc/$VERSION-*/php.ini-development /etc/php.ini
 	sed -i "s/;date\.timezone.*/date\.timezone = ${TZ}/g" /etc/php.ini
 	sed -i "s/memory_limit.*/memory_limit = ${MEMORY_LIMIT}M/g" /etc/php.ini
+	sed -i "s/max_execution_time.*/max_execution_time = ${EXEC_TIME}/g" /etc/php.ini
```
Comment (willmorgan) [inline scripts/php.sh]: I think like #106 and as mentioned in #84, we should be echoing out the fact that we are potentially overwriting PHP settings here. Thoughts?

File: scripts/php.sh (hunk 1 above)
Comment (dhensby) [inline scripts/php.sh]: that's some serious scope creep

Comment (willmorgan) [inline scripts/php.sh]: :heart:

File: scripts/php.sh (hunk 2 above)
Comment (dhensby) [inline scripts/php.sh]: I don't think so.

1. It sets it to 60 seconds, a sensible default for a vagrant box IMO
2. The dev can change this via the `-e` flag (why warn them of something they did on purpose)
3. If we warn on this then why not every setting
4. The warnings are just going to get lost in a ton of output so I doubt anyone will notice it anyway

Comment (dhensby) [inline scripts/php.sh]: What may be better is to only update the value if it's currently less than the value we're looking to move to? But then a dev can't reduce the time via this script (which is fine, that's not it's purpose, I suppose - it's just for initial set-up of PHP)

Comment (willmorgan) [inline scripts/php.sh]: Yeah, fair enough.

**Comment by dhensby** [comment]
boom
